components/brave_rewards/browser/rewards_p3a.h

```diff
@@ -73,6 +73,19 @@ void RecordNoWalletCreatedForAllMetrics();
 
 void RecordRewardsDisabledForSomeMetrics();
 
+enum class RewardsEnabledDuration {
+  kNever,
+  kStillEnabled,
```

Comment:
This value seems a bit out of place in this enumeration, as it doesn't represent a duration per se. Does this mean that it's of unknown duration? If so, maybe its name should reflect that.

Reply:
There are two values representing corner cases where we can't measure a duration. Both are specified as answers in the original P3A question:

- Rewards have never been enabled
- Rewards are currently (still) enabled

We're asking how long rewards were enabled for after someone turns them off. So it's not that the duration is unknown in these states, it's that the question doesn't apply.

Are you asking for something like `RewardsEnabledDuration::kUnknownBecauseNotDisabled`? I was trying to keep the length similar to the duration values, while keying on the 'enabled' part of the type name.

Reply:
Gotcha, I think that's fine then.